src/components/Files/FileManagerModal.tsx

```diff
@@ -354,7 +354,9 @@ export const FileManagerModal = ({
                   {t(
                     'Max file size up to 512 Mb. Supported types: {{allowedExtensions}}.',
                     {
-                      allowedExtensions: allowedExtensions.join(', '),
+                      allowedExtensions:
+                        allowedExtensions.join(', ') ||
+                        'no available extensions',
```

Comment:
Maybe should be ` all` by default?

Reply:
For all types we have special '*/*' case, so this is not real case, only when incorrect configuration